config/api/assessment_schema.json

```diff
@@ -0,0 +1,382 @@
+{
+  "id": "#{Rails.root}/config/api/assessment_schema.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Legal Aid Check Financial Eligibility payload schema",
+  "description": "This schema defines the payload required to submit data to the Legal Aid Check Financial Eligibility API",
+
+
+  "type": "object",
+  "required": [
+    "assessment_id",
+    "meta_data",
+    "applicant",
+    "applicant_income",
+    "applicant_outgoings",
+    "applicant_capital"
+  ],
+  "additionalProperties": false,
+
+
+  "definitions": {
+    "uuid": {
+      "type": "string",
+      "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$",
+      "description": "has to be a valid uuid"
+    },
+    "date": {
+      "type": "string",
+      "pattern": "^([12][0-9]{3}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01]))$",
+      "description": "Date in format YYYY-MM-DD in range 1900-01-01 to 2999-12-31"
+    },
+    "matter_proceeding_type": {
+      "type": "string",
+      "pattern": "^Domestic abuse$",
```

Comment:
Should this be an enum with only one member?

Reply:
yep - we're going to be adding more, so makes sense.